gobase/command/db/drop.go

```diff
@@ -25,21 +26,31 @@ func GetAllTables(db *sql.DB) string {
 		err = rows.Scan(
 			&table,
 		)
-		if len(tables) == 0 {
-			tables = table
+
+		if err != nil {
+			fmt.Println(err)
+		}
+
+		if stringBuilder.Len() == 0 {
+			stringBuilder.WriteString("\"" + table + "\"")
 		} else {
-			tables = tables + "," + table
+			stringBuilder.WriteString("," + "\"" + table + "\"")
 		}
 	}
-	return tables
+	return stringBuilder.String()
 }
 
-func Drop(db *sql.DB) {
-	tables := GetAllTables(db)
+func Drop(db *sql.DB, domainName string) {
+	var tables string
+	if domainName == "all" {
+		tables = GetAllTables(db)
+	} else {
```

Comment:
Try to test if the user misstype table name. If the information is not right need to create a handler to provide good information to user

Reply:
`query := "DROP TABLE " + tables
	_, err := db.Exec(query)
	if err != nil {
		log.Fatal(err)
	}`
The error will be handled in here, i.e if we misstype the table name it will show an error message that the table doesn't exist

Reply:
kay